packages/eslint-config/index.js

```diff
@@ -121,7 +121,12 @@ module.exports = {
     'import/namespace': 'error',
     'import/no-absolute-path': 'error',
     'import/no-dynamic-require': 'error',
-    'import/no-internal-modules': 'error',
+    'import/no-internal-modules': [
```

Comment:
This is something for discussion, since this is the config we also ship to customers. Should costomers also do not get a warning if they include internal modules and use our linter? For me this makes sense.